components/global/global_header.tsx

```diff
@@ -41,27 +26,49 @@ const GlobalHeader = () => {
         return null;
     }
 
+    const theme = {
+        ...lightTheme,
+        noStyleReset: true,
+        noFontFaces: true,
+        noDefaultStyle: true,
+        background: {
+            ...lightTheme.background,
+            shape: 'var(--sidebar-teambar-bg)',
+        },
+    };
+
+    // adding the dark theme for now to test how the colors "perform"
     return (
-        <HeaderContainer>
-            <ProductSwitcher/>
-            <AppSpectificContent>
-                {currentProductID !== null &&
-                    <Pluggable
-                        pluggableName={'Product'}
-                        subComponentName={'headerComponent'}
-                        pluggableId={currentProductID}
+        <>
+            <ThemeProvider theme={theme}>
```

Comment:
I am assuming this is temporary and will eventually be specified at a higher level?

Reply:
Absolutely